ci/docker/kpsrbase-1.0.0.dockerfile

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:20.04
+
+ENV TZ 'Europe/Madrid'
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -y update && \
+    apt-get -y install \
+    autoconf \
+    build-essential \
+    ccache \
+    clang-format-11 \
+    clang-tidy-11 \
+    cmake \
+    cppcheck \
+    curl \
+    doxygen \
+    doxygen-doc \
+    gcovr \
+    git \
+    gnupg2 \
+    graphviz \
+    lcov \
+    libtool \
+    locales \
+    locales-all \
+    ninja-build \
+    pkg-config \
+    python \
+    python3-pip \
+    python3-numpy \
```

Comment:
add python3-venv so that we can update the python install method for code generator